tests/mappings/filter_directory/BUILD

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@rules_python//python:defs.bzl", "py_test")
 load(":defs.bzl", "inspect_directory_test")
+load("//pkg:mappings.bzl", "filter_directory")
 load("//tests/util:defs.bzl", "directory")
-load("//:mappings.bzl", "filter_directory")
+load("@rules_python//python:defs.bzl", "py_test")
 
```

Comment:
I think `buildifier` will want to see the rules_python one first.

Reply:
It doesn't seem to care.  It is strident about so many other things, yet it does not care about something easy and, by definition, reorderable like this one.